give_kudos.py

```diff
@@ -38,52 +37,67 @@ def email_login(self):
         self.page.fill("#password", self.PASSWORD)
         self.page.click("button[type='submit']")
         print("---Logged in!!---")
-        self.page.goto(os.path.join(BASE_URL, "dashboard"), wait_until="domcontentloaded")
-        
+        # limit activities count by GET parameter
+        self.page.goto(os.path.join(BASE_URL, "dashboard?num_entries=50"), wait_until="networkidle")

```

Comment:
Let's set the number of entries as a class variable in `__init__`. I would prefer to set this to a high number like 200 so that we can run this script fewer times and "get more done". e.g. 
```suggestion
        self.page.goto(os.path.join(BASE_URL, f"dashboard?num_entries={self.num_entries}"), wait_until="networkidle")
```

Reply:
done